### PR TITLE
Fix channels_last for LightningModule

### PR DESCRIPTION
**Pull request by y199387**
## Description
Fix bug in channels_last.

### How to test?
- [x] Unit test

### CODE REVIEW

**Comment by yangw1234** [comment]
maybe add an unit test to cover the bug

**Comment by y199387** [comment]
> maybe add an unit test to cover the bug

Sure

File: python/nano/test/pytorch/tests/test_channels_last.py [hunk 1]
```diff
@@ -17,66 +17,97 @@
...
+
 num_classes = 10
 batch_size = 32
 dataset_size = 256
 num_workers = 0
 data_dir = os.path.join(os.path.dirname(__file__), "../data")
 
-class customResNet(ResNet):
+
+class customResNet(pl.LightningModule):
```
Comment (yangw1234) [inline python/nano/test/pytorch/tests/test_channels_last.py:38]: Class name should generally start with an upper case letter.

**Comment by yangw1234** [comment]
LGTM

**Comment by yangw1234** [comment]
we also need jenkins

**Comment by yangw1234** [comment]
http://10.112.231.51:18889/view/BigDL-PR-Validation/job/ZOO-PR-NanoTests/342/

**Pull request merged**